en/migrate-tidb-to-kubernetes.md

```diff
@@ -0,0 +1,119 @@
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    search default.svc.cluster.local svc.cluster.local cluster.local  
+    nameserver <CoreDNS Pod_ip_1>
+    nameserver <CoreDNS Pod_ip_2>
+    nameserver <CoreDNS Pod_ip_n>
+    ```
+
+3. Test whether the node can successfully parse the internal IP address in Kubernetes:
+
+    ```shell
+    $ ping basic-pd-2.basic-pd-peer.blade.svc
+    PING basic-pd-2.basic-pd-peer.blade.svc (10.24.66.178) 56(84) bytes of data.
+    64 bytes from basic-pd-2.basic-pd-peer.blade.svc (10.24.66.178): icmp_seq=1 ttl=61 time=0.213 ms
+    64 bytes from basic-pd-2.basic-pd-peer.blade.svc (10.24.66.178): icmp_seq=2 ttl=61 time=0.175 ms
+    64 bytes from basic-pd-2.basic-pd-peer.blade.svc (10.24.66.178): icmp_seq=3 ttl=61 time=0.188 ms
+    64 bytes from basic-pd-2.basic-pd-peer.blade.svc (10.24.66.178): icmp_seq=4 ttl=61 time=0.157 ms
+    ```
+
+## Step 2: Create a TiDB cluster in Kubernetes
+
+1. Get the PD node address and port of the source cluster via [PD Control](https://docs.pingcap.com/tidb/stable/pd-control):
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    pd-ctl -u http://<address>:<port> member | jq '.members | .[] | .client_urls'
+    ```
+
+2. Create the target TiDB cluster in Kubernetes, which should have at least 3 TiKV nodes. Specify the PD node address of the source cluster in the `spec.PDAddresses` field (starting with `http://`):
+
+    ```yaml
+    spec
+      ...
+      PDAddresses:
+      - http://pd1_addr:port
+      - http://pd2_addr:port
+      - http://pd3_addr:port
+    ```
+
+3. Confirm that the source cluster and the target cluster compose of a new cluster that runs normally:
+
+    - Get the the number and state of stores in the new cluster:
+
+        {{< copyable "shell-regular" >}}
+        
+        ```shell
+        # Get the number of stores
+        pd-ctl -u http://<address>:<port> store | jq '.count'
+        # Get the state of stores
+        pd-ctl -u http://<address>:<port> store | jq '.stores | .[] | .store.state_name'   
+        ```
+    
+    - [Access the TiDB cluster in Kubernetes](access-tidb.md) via MySQL client.
+
+## Step 3: Scale in the TiDB nodes of the source cluster
+
+Remove all TiDB nodes of the source cluster:
+
+- If the source cluster is deployed using TiUP, refer to [Scale in a TiDB cluster](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup#scale-in-a-tidbpdtikv-cluster).
+
+- If the source cluster is deployed using TiDB Ansible, refer to [Decrease the capacity of a TiDB node](https://docs.pingcap.com/tidb/stable/scale-tidb-using-ansible#decrease-the-capacity-of-a-tidb-node).
+
+> **Note:**
+>
+> If you access the source TiDB cluster via load balancing or database middleware, you need to first modify the configuration to route your application traffic to the target TiDB cluster. Otherwise, your application might be affected.
```

Comment:
```suggestion
> If you access the source TiDB cluster via load balancer or database middleware, you need to first modify the configuration to route your application traffic to the target TiDB cluster. Otherwise, your application might be affected.
```